server/src/lib/db.ts

```diff
@@ -30,6 +30,7 @@ export const authenticateUser = (
     webId: any,
     login: any,
     name: any,
+    clientCode: any,
 ): Promise<void> => new Promise((resolve: () => void, reject: Rejector) => {
```

Comment:
About the method name: isn't this method more like a `createUser` than a `authenticateUser`, this method does pretty much nothing with authenticating, it only creates/updates a user in the db with the given parameters